apiserver/facades/controller/instancepoller/service_mock_test.go

```diff
@@ -0,0 +1,699 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Delete this file. There is no instance poller facade.

Reply:
Sure, bad rebase. Removed.